### PR TITLE
Translate block labels

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

File: Block/Social/TwitterEmbedTweetBlockService.php [hunk 1]
```diff
@@ -76,12 +76,12 @@ public function configureSettings(OptionsResolver $resolver)
...
             'template'    => 'SonataSeoBundle:Block:block_twitter_embed.html.twig',
             'tweet'       => '',
-            'maxwidth'    => null,      // Should be between 250 and 550px
+            'maxwidth'    => null, // Should be between 250 and 550px
             'hide_media'  => false,
             'hide_thread' => false,
-            'omit_script' => false,     // Should be asked for only once in a page
-            'align'       => 'none',    // Left, right, center or none
-            'related'     => null,      // Accounts related to content
+            'omit_script' => false, // Should be asked for only once in a page
```
Comment (OskarStark) [inline Block/Social/TwitterEmbedTweetBlockService.php]: what about using this comments for help messages?

File: Resources/translations/SonataSeoBundle.de.xliff [hunk 2]
```diff
@@ -54,6 +54,238 @@
 				<source>sonata.seo.block.pinterest.pin_button</source>
 				<target>Pinterest - Pin</target>
 			</trans-unit>
```
Comment (OskarStark) [inline Resources/translations/SonataSeoBundle.de.xliff]: indentions looks weird

File: Block/Social/TwitterEmbedTweetBlockService.php (hunk 1 above)
Comment (core23) [inline Block/Social/TwitterEmbedTweetBlockService.php]: Would create a separate PR for this.

Comment (OskarStark) [inline Block/Social/TwitterEmbedTweetBlockService.php]: :+1:

**Comment by OskarStark** [comment]
Thank you @core23

**Pull request merged**